src/browser/api/window.js

```diff
@@ -2165,4 +2165,8 @@ function getElectronBrowserWindow(identity, errDesc) {
     return browserWindow;
 }
 
+function getIdentifier(preload) {
+    return preload.url ? preload.url : `${preload.name}-${preload.version}`;
```

Comment:
@VanessaSergeon why is getIdentifier also defined here ?

Reply:
Oh thanks for catching that duplication. Will import from preload_scripts.ts instead.